test/sharedtests/mcd_config_drift.go

```diff
@@ -0,0 +1,396 @@
+package sharedtests
```

Comment:
Request: retitle dir to use "e2e" like e2e-shared-tests also use convention of _test.go in the file name.  Current naming is unclear. 

Might also be nice to add a README to your dir to specific what you mean by shared tests- these are shared between e2e and single-node and that's the purpose of the dir (correct me if im wrong on that ;) )

Reply:
You're correct! I was struggling with a good name for these, so that helps.

Reply:
I can't use the `_test.go` convention in the filename, otherwise I get:

```console
go build github.com/openshift/machine-config-operator/test/e2e-shared-tests: no non-test Go files in /Users/zzlotnik/go/src/github.com/openshift/machine-config-operator/test/e2e-shared-tests
```

Reply:
ahhhhhh gotcha